homeassistant/components/homekit_controller/binary_sensor.py

```diff
@@ -113,18 +113,12 @@ class HomeKitBatteryLowSensor(HomeKitEntity, BinarySensorEntity):
 
     _attr_device_class = BinarySensorDeviceClass.BATTERY
     _attr_entity_category = EntityCategory.DIAGNOSTIC
+    _attr_name = "Low Battery"
```

Comment:
I think the policy is to capitalise only the first word.
```suggestion
    _attr_name = "Low battery"
```